uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/manager-leave/app.js

```diff
@@ -28,8 +28,10 @@ define([
     'leave-absences.directives',
     'leave-absences.models',
   ])
-  .run(['$log', function ($log) {
+  .run(['$log', '$rootScope', 'settings', function ($log, $rootScope, settings) {
     $log.debug('app.run');
+
+    $rootScope.pathTpl = settings.pathTpl;
```

Comment:
For `my-leave` you needed `shared-settings`, so why do you need only `settings` here?